raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -25,7 +24,6 @@ nodes:
 
   default_options:
     gas-price: fast
-    environment-type: development
```

Comment:
Wouldn't it make sense to be explicit about this and define it here as a template expression instead of "magically" assigning the option in the SP?

Reply:
I was afraid that people write scenarios without that part and get mismatches between what they configure in the environment and actually gets executed.

Reply:
Hm, that is a concern but AFAICS it does apply to every other setting coming from the environment file as well, doesn't it?

I think the scenario file should be the 'source of truth' (and if placeholders are involved they should be explicit), but we don't have to discuss this here and now.

Let's merge it and come back to this later.

Reply:
Yes, there were quite a lot of small decisions to make when introducing this. I certainly didn't find the perfect choice for all of them. But it should be a nice foundation upon which we can clean things up.